packages/cf-component-box/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cf-component-box",
   "description": "Cloudflare Box Component",
-  "version": "1.4.3",
+  "version": "2.0.0",
```

Comment:
Yup. I was pulling from `origin`(my fork remote) and not `upstream`(this remote).